tests/gtests/graph/unit/utils.hpp

```diff
@@ -1165,6 +1161,169 @@ inline void construct_select_float_MHA(dnnl::impl::graph::graph_t *agraph,
     agraph->add_op(&reorder_output);
 }
 
+inline void construct_select_float_MHA_without_broadcast(
+        dnnl::impl::graph::graph_t *agraph,
+        impl::data_type_t dtype = impl::data_type::f32, int batch_size = 1,
+        int seq_len = 128, int num_head = 12, int head_dim = 768,
+        bool transpose = false, bool distil = true, bool gpt = false) {
```

Comment:
If this is needed, it can be incorporated into the function above, probably with a flag in the parameter:
```suggestion
        bool transpose = false, bool distil = true, bool gpt = false, bool bcast_select = true) {
```